src/CMakeLists.txt

```diff
@@ -93,8 +93,8 @@ if (LPM_USE_FFTW3)
   target_link_libraries(lpm PUBLIC ${FFTW_LIBRARIES})
 endif()
 if (LPM_USE_FINUFFT)
-  target_link_libraries(lpm PUBLIC finufft_static)
-  add_link_options(-fsanitize=address)
```

Comment:
Is this sufficient to close #112?

Reply:
Added discussion to #112.   Not ready to close it yet; for now, DFS only works with non-debug builds.